addons/payment_worldline/tests/test_worldline.py

```diff
@@ -0,0 +1,123 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from base64 import b64encode
+import hashlib
+import hmac
+import json
+from unittest.mock import patch
+
+from werkzeug.exceptions import Forbidden
+
+from odoo.tests import tagged
+from odoo.tools import mute_logger
+
+from odoo.addons.payment.tests.http_common import PaymentHttpCommon
+from odoo.addons.payment_worldline.controllers.main import WorldlineController
+from odoo.addons.payment_worldline.tests.common import WorldlineCommon
+
+
+@tagged('post_install', '-at_install')
+class WorldlineTest(WorldlineCommon, PaymentHttpCommon):
```

Comment:
Let's keep it as-is for now, but it'd be great to split the tests to easy their understanding and maintenance. See [task 3054481](https://www.odoo.com/odoo/action-1592/new/4106/tasks/3054481).